$OpenBSD: patch-sslscan_c,v 1.7 2017/01/27 11:01:14 sthen Exp $

use the accessor function; this struct member was made internal-only in libressl

--- sslscan.c.orig	Fri Jan 27 10:55:06 2017
+++ sslscan.c	Fri Jan 27 10:56:26 2017
@@ -1155,14 +1155,14 @@ int testRenegotiation(struct sslCheckOptions *options,
                                 printf_verbose("Attempting SSL_do_handshake(ssl)\n");
                                 SSL_do_handshake(ssl); // Send renegotiation request to server //TODO :: XXX hanging here
 
-                                if (ssl->state == SSL_ST_OK)
+                                if (SSL_get_state(ssl) == SSL_ST_OK)
                                 {
                                     res = SSL_do_handshake(ssl); // Send renegotiation request to server
                                     if( res != 1 )
                                     {
                                         printf_error("\n\nSSL_do_handshake() call failed\n");
                                     }
-                                    if (ssl->state == SSL_ST_OK)
+                                    if (SSL_get_state(ssl) == SSL_ST_OK)
                                     {
                                         /* our renegotiation is complete */
                                         renOut->supported = true;
